src/types.js

```diff
@@ -1,15 +1,62 @@
 // @flow
 
 export type TXOBJ = {
-  locktime: number,
+  locktime?: number,
   version: number,
   ins: {
     output: { hash: string, vout: number },
     script: string,
     sequence: string,
     prevScriptPubKey: string,
   }[],
-  outs: { script: string, satoshis: number }[],
+  outs: { script: string, satoshis: number, isFromBackwardTransfer?: boolean }[],
+  sc_params?: {
```

Comment:
@loucaponi Thoughts on using snake_case here or should we make everything camelCase?

Reply:
I prefer camel case :)

Reply:
I'm not complete sure about this, i used the same fields names used by zendoo in order to avoid confusion (also the bitcore-lib-zen use this notation). Even if we decide to rename them we have to do also in the bitcore-lib-zen.

Reply:
@MarcoOl94 I understand consistency with Zendoo. 

Why does the naming have to be the same between zencashjs and bitcore-lib-zen? Is it so we can seamlessly replace bitcore-lib-zen with zencashjs?

Reply:
I don't think we can replace the bitcore-lib-zen right now. I mean maybe in the mongo version of the explorer but i don't know if we have time to replace the bitcore-lib-zen also in the no-mongo explorer.

Also keep in mind that Sphere should use the same interface both in full mode (zendoo) and light mode (explorer+zencashjs).

Reply:
agree with Marco: we should mantain if possible the same output of the rpc commands, so will be easy for Sphere  and maybe other clients in the future to switch fullmode<->lightmode